migtests/tests/analyze-schema/dummy-export-dir/schema/tables/table.sql

```diff
@@ -24,3 +24,21 @@ CREATE TABLE t1 (
 	descriptions varchar(50),
 	PRIMARY KEY (id)
 ) PARTITION BY LIST (country_code, record_type) ;
+
+CREATE TABLE test_1 (
```

Comment:
Whenever we add any DDL statements to the automation, add a comment stating what aspect of the DDL statement is important. e.g. here you can mention that this is a table that has no primary key and is partitioned by a NOT NULL attribute. Such a table should not be flagged by the analyze-schema.